src/client/components/app/Transactions/DeployLineTx.tsx

```diff
@@ -64,8 +64,16 @@ export const DeployLineTx: FC<DeployLineProps> = (props) => {
   };
 
   const onAmountChange = (ttl: string) => {
-    let timeToLive = +ttl * 24 * 60 * 60;
-    setTimeToLive(timeToLive.toString());
+    if (Number(ttl) < 0) {
+      setTTLWarning('Increase TTL, cannot be 0.');
+      return;
+    }
+    if (Number(ttl) > 0) {
```

Comment:
what exactly are we testing here?

Reply:
if Number(ttl) > 0   
we remove the warning.